Noisy imagenette more info in new folder

tmabraham opened the pull request:
@hamelsmu: As we discussed on Discord, I have added a separate folder with more information about the dataset (label generation, extended LB). 

Should I also edit the Imagenette README mentioning the update to the Imagenette dataset and introducing the noisy Imagenette dataset/LB? Or will that be done later?

hamelsmu commented:
Yeah I think updating the README is a good idea.  Make sure to mention which information is optional given we are only showing certain noise levels etc

tmabraham commented:
@hamelsmu Let me know if this is fine, or if you think something needs to be more clear...

hamelsmu commented on README.md:
```diff
@@ -39,6 +42,8 @@ Imagewoof is a subset of 10 classes from Imagenet that aren't so easy to classif
 - [320 px download](https://s3.amazonaws.com/fast-ai-imageclas/imagewoof2-320.tgz); 
 - [160 px download](https://s3.amazonaws.com/fast-ai-imageclas/imagewoof2-160.tgz). 
 
+The dataset also comes with a CSV file with 1%, 5%, 25%, and 50% of the labels randomly changed to an incorrect label. More information about the noisy labels are provided in the "noisy_imagenette" folder.
```
consider renaming the folder to `noisy_labels` since your thing applies to both imagenette and imagwoof

tmabraham replied:
@hamelsmu Good point, thanks for the catch! I have fixed it now.

hamelsmu reviewed (commented):
Minor request

hamelsmu reviewed (changes requested):
I think it would be good to add a bit more prose/ commentary in the notebook, as this might be viewed by others.

tmabraham commented:
@hamelsmu Thank you for your suggestions! I have added more detail to `generate_labels.ipynb`, renamed the folder, and updated the READMEs. Let me know if there are more changes you think need to be made...

hamelsmu commented:
LGTM
